File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        
        int[] answer = new int[commands.length];
        
        for (int i = 0; i < commands.length; i++) {
            int start = commands[i][0]; // i
            int end = commands[i][1];   // j
            int k = commands[i][2];     // k
            
            int[] elements = Arrays.copyOfRange(array, start - 1, end);
            
            Arrays.sort(elements);
            
            answer[i] = elements[k - 1];
        }
        
        return answer;
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for arr in commands:
        i, j, k = arr[0], arr[1], arr[2]
        
        array_cutting = []
        
        for n in range(i-1, j):
            array_cutting.append(array[n])
        
        array_cutting.sort()
        answer.append(array_cutting[k-1])

        
    
    return answer